package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "p2p-graph",
   "description": "Real-time P2P network visualization with D3",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Btw, these terminal aliases might come in handy for you. https://github.com/feross/prezto/blob/master/runcoms/zshrc#L30-L34

Then you can just run `patch`, `minor`, or `major` to prepare and release a new version.